test/simplify_algebra_test.cpp

```diff
@@ -222,8 +222,8 @@ TEST_CASE(simplify_mul_conv1)
     mm->add_instruction(pass_op{}, mul);
     EXPECT(conv->outputs().front()->name() == "mul");
     run_pass(p);
-    auto new_conv =
-        std::find_if(p.begin(), p.end(), [](auto&& ins) { return ins.name() == "convolution"; });
+    auto new_conv = std::find_if(
+        mm->begin(), mm->end(), [](auto&& ins) { return ins.name() == "convolution"; });
```

Comment:
These tests should just run over a module. There is no need to create a program. Maybe we can fix this at a later time.